modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2806,6 +2806,38 @@ definition {
 		}
 	}
 
+	@description = "LPS-176829 - Verify that Formula fields can't be required"
+	@priority = 4
+	test FormulaFieldsIsNotMandatory {
+		property portal.acceptance = "true";
+
+		task ("Given: an Object") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object test",
+				objectName = "CustomObjecttest",
+				pluralLabelName = "Custom Objects test");
```

Comment:
In the `addObjectviaAPI`, we use the `Custom Object + LPS from the automation ticket`, this way we avoid errors in the CI due to duplicate objects.

Ex:
```
ObjectAdmin.addObjectViaAPI(
	labelName = "Custom Object 176829",
	objectName = "CustomObject176829",
	pluralLabelName = "Custom Objects 176829");
```